ui/component/walletSwap/view.jsx

```diff
@@ -421,7 +420,6 @@ function WalletSwap(props: Props) {
               label={__('Copy transaction ID')}
               title={sendTxId}
               onClick={() => {
-                clipboard.writeText(sendTxId);
```

Comment:
Hm ... was this broken for Web all along?  How was Web copying it to the clipboard?
Hard to test now given Swap is disabled ...

Reply:
Don't care for now since Swapping has been disabled.